osx/Keybase.xcodeproj/project.pbxproj

```diff
@@ -1274,7 +1345,7 @@
 				CLANG_WARN_SUSPICIOUS_MOVE = YES;
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
-				CODE_SIGN_IDENTITY = "-";
+				CODE_SIGN_IDENTITY = "Mac Developer";
```

Comment:
I guess this doesn't matter since we have a separate codesign process. But thought I'd bring it up just to make sure.